### DISCUSSION TITLE
LoadLibrary failed with error 126 - onnxruntime_providers_cuda.dll

### DISCUSSION THREAD

**Discussion by GregMalick** [Comfy-Org/ComfyUI#2510] [Q&A]
I have comfyUI installed & running fine on my internal SSD "J" drive. Works pretty much perfectly.
I've also installing comfyUI on a removable SSD "X" drive so I can take comfyUI to work with me.
The X drive works as far as I've tested except for this error message:
FAIL : LoadLibrary failed with error 126 "" when trying to load "X:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\onnxruntime\capi\onnxruntime_providers_cuda.dll"
Both drives have the **onnxruntime_providers_cuda.dll**  in: \ComfyUI_windows_portable\python_embeded\Lib\site-packages\onnxruntime\capi\

Both dlls are the same size.   
Any ideas/suggestions?

**Comment by fogostudio** [comment]
same error here

**Reply by GregMalick** [reply]
Everything seems to work.  Just hate unexplained errors like this.
